Modules/Module_Processing/xASL_init_LoadMetadata.m

```diff
@@ -133,7 +133,7 @@
         end
 
         for iSession=1:x.dataset.nSessions
-            VarDataOri([iSession:x.dataset.nSessions:x.dataset.nSubjectsSessions-x.dataset.nSessions+iSession], 1) = x.SUBJECTS(:);
+            VarDataOri(iSession:x.dataset.nSessions:(x.dataset.nSubjects * x.dataset.nSessions - x.dataset.nSessions + iSession), 1) = x.SUBJECTS(:);

```

Comment:
Same here, we may need the square brackets

Reply:
We don't . As above, we only need square brackets to construct a list from several variables.
The : sign constructs a list by itself and the [] brackets are not needed.